user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently Asked Questions
+
+## Can I Swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+As of DataMiner 10.5.1, only basic elements are supported.
+
+## Does the ID of an element change when it is Swarmed?
+
+No, the identifier of the element stays the same.
+
+Each element also stores where it is hosted, that field is updated when Swarmed.
+
+## Can I block an element from Swarming?
+
+Not in the initial release but is expected to be available in DataMiner 10.5.5/10.6.0.
+
+## What does the **DataMinerID** mean of an element that Swarmed?
```

Comment:
Not sure if this is clear enough. I had some trouble reading this but maybe I have been used to using the dmaid/eid identifier pair for too long

Reply:
Not sure, it is confusing to explain. I'll see what docs team has to say. We are very biased here